docs/data/quality-metrics.md

```diff
@@ -0,0 +1,80 @@
+---
+id: qc-metrics
+title: Quality Control Metrics
+sidebar_label: QC Metrics
+---
+
+ICGC ARGO provides access to data from ARGO member programs generated through the standardized [ARGO Analysis pipelines](/docs/analysis_workflows/data-types). The following page lists quality control metric analyses and data files generated by the ARGO Analysis pipelines.
```

Comment:
should it be

quality control metric `analysis` and data files

Reply:
or is this plural?

Reply:
its plural